iset.mm

```diff
@@ -136047,6 +136061,163 @@ S C_ ( P ( ball ` D ) T ) ) $=
+      XNBUMZXSAUMZMZYNXJEJYSXRWKYMXIYSXPWIXQWJYSXOWHJXNBYQYRUTZYSXNBYTVAZVBYSXO
+      WHJUUAVCVDYSXTWLYLXHYSXSAJYQYRVIZVEYSYKXGFTYSYJXFDTYSYIXECXOWHUUAYSYEWSYH
+      XDYSYAWNYDWRYSXSAWMUFUUBVFYSYCWPWQRYSYBWOPYSXSAWMOUUBVGVJVKVDYSYGXBXCRYSY
+      FWTXAPOYSWMXNBYTVLVMVKVNVOVPVQVDVDVRYPVSVTWAXJEWFJWBWCWD $.
+  $}
+
+  ${
+    $d A z $.  $d B f u w x y z $.  $d C u x y z $.  $d F f u w x y z $.
+    $d ph x y $.
+    ellimc3.f $e |- ( ph -> F : A --> CC ) $.
+    ellimc3.a $e |- ( ph -> A C_ CC ) $.
+    ellimc3.b $e |- ( ph -> B e. CC ) $.
+    $( Write the epsilon-delta definition of a limit.  (Contributed by Mario
+       Carneiro, 28-Dec-2016.)  Use apartness.  (Revised by Jim Kingdon,
+       3-Jun-2023.) $)
+    ellimc3ap $p |- ( ph -> ( C e. ( F limCC B ) <-> ( C e. CC /\
+      A. x e. RR+ E. y e. RR+ A. z e. A
+        ( ( z =//= B /\ ( abs ` ( z - B ) ) < y ) ->
+          ( abs ` ( ( F ` z ) - C ) ) < x ) ) ) ) $=
+      ( vu wcel cc wa cmin cabs clt wral crp vf vw climc co cdm wss cap wbr cfv
+      wf cv wi wrex crab cpm wceq cnex elpm2r syl22anc rabex simpl dmeqd feq12d
+      cvv a1i sseq1d anbi12d eleq1d breq2d oveq2d fveq2d breq1d fveq1d fvoveq1d
+      simpr imbi12d raleqbidv rexbidv ralbidv rabbidv df-limced ovmpt2ga eleq2d
+      syl3anc oveq2 imbi2d anbi2d elrab syl6bb biantrurd wb fdmd mpbird eqsstrd
+      feq2d ibar syl2anc bitrd raleqdv 3bitr2d ) AGHFUCUDZMZGNMZHUEZNHUJZXDNUFZ
+      OZFNMZDUKZFUGUHZXIFPUDZQUIZCUKZRUHZOZXIHUIZGPUDZQUIZBUKZRUHZULZDXDSZCTUMZ
+      BTSZOZOZOZXCYDOXCYADESZCTUMZBTSZOAXBGXGXHXOXPLUKZPUDZQUIZXSRUHZULZDXDSZCT
+      UMZBTSZOZOZLNUNZMYGAXAUUAGAHNNUOUDZMZXHUUAVDMZXAUUAUPANVDMZUUEENHUJZENUFU
+      UCUUEAUQVEZUUGIJNNEHVDVDURUSKUUDAYTLNUQUTVEUAUBHFUUBNUAUKZUEZNUUHUJZUUINU
+      FZOZUBUKZNMZXIUUMUGUHZXIUUMPUDZQUIZXMRUHZOZXIUUHUIZYKPUDQUIZXSRUHZULZDUUI
+      SZCTUMZBTSZOZOZLNUNUUAUCVDUUHHUPZUUMFUPZOZUVHYTLNUVKUULXGUVGYSUVKUUJXEUUK
+      XFUVKUUIXDNUUHHUVIUVJVAZUVKUUHHUVLVBZVCUVKUUIXDNUVMVFVGUVKUUNXHUVFYRUVKUU
+      MFNUVIUVJVOZVHUVKUVEYQBTUVKUVDYPCTUVKUVCYODUUIXDUVMUVKUUSXOUVBYNUVKUUOXJU
+      URXNUVKUUMFXIUGUVNVIUVKUUQXLXMRUVKUUPXKQUVKUUMFXIPUVNVJVKVLVGUVKUVAYMXSRU
+      VKUUTXPYKQPUVKXIUUHHUVLVMVNVLVPVQVRVSVGVGVTUBLDBUACWAWBWDWCYTYFLGNYKGUPZY
+      SYEXGUVOYRYDXHUVOYQYCBTUVOYPYBCTUVOYOYADXDUVOYNXTXOUVOYMXRXSRUVOYLXQQYKGX
+      PPWEVKVLWFVSVRVSWGWGWHWIAYDYFXCAYDYEYFAXHYDKWJAXEXFYEYFWKAXEUUFIAXDENHAEN
+      HIWLZWOWMAXDENUVPJWNXGYEWPWQWRWGAYDYJXCAYCYIBTAYBYHCTAYADXDEUVPWSVRVSWGWT
+      $.
+  $}
+
+  ${
+    $d A d e u x z $.  $d B d e u x z $.  $d F d e u z $.  $d d e ph u z $.
+    limccl.f $e |- ( ph -> F : A --> CC ) $.
+    limcdifap.a $e |- ( ph -> A C_ CC ) $.
+    $( It suffices to consider functions which are not defined at ` B ` to
+       define the limit of a function.  In particular, the value of the
+       original function ` F ` at ` B ` does not affect the limit of ` F ` .
+       (Contributed by Mario Carneiro, 25-Dec-2016.) $)
+    limcdifap $p |- ( ph ->
```

Comment:
Shpould also be marked with"Revised by...".

Reply:
Because of changing not equal to apart, or because of the ` A C_ CC ` hypothesis? A lot of these attributions seem pretty wordy to me, especially if the "short description of the revision" mentioned in https://us.metamath.org/mpeuni/conventions-comments.html is taken to be something we always do.

I think there is some desire to keep these brief (as opposed to the full history that something went through) but I never know how far to take that.

Reply:
Also here, adding "(Revised by Jim Kingdon, 3-Jun-2023.)" would be sufficient.

Reply:
Done. I wonder if we should make the conventions describe the short description as more of an option than something that looks like a requirement, though.

Reply:
I don't think explaining revisions should be necessary, it's documentation not a changelog. The docs should reflect the current version of the definition and should be revised if they are out of date, but otherwise the acknowledgement of the change itself should be good enough.

Reply:
I have made https://github.com/metamath/set.mm/pull/3240 where we can discuss what we want to say in conventions.